### PR TITLE
Blog post about new RFC process

### PR DESCRIPTION
**Pull request by acdlite**
Will publish tomorrow morning

### CODE REVIEW

File: content/blog/2017-12-07-introducing-the-react-rfc-process.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+We're adopting an RFC ("request for comments") process for contributing ideas to React. 
+
+Inspired by [Yarn](https://github.com/yarnpkg/rfcs), [Ember](https://github.com/emberjs/rfcs), and [Rust](https://github.com/rust-lang/rfcs), the goal is to allow React core team members and community members to collaborate on the design of new features. It's also intended to provide a clear path for ideas to enter the project:
+
+- Create an RFC document detailing your proposal.
+- Submit a PR to the [RFC repository](https://github.com/reactjs/rfcs).
+- Incorporate feedback into the proposal.
+- After discussion, the RFC may be accepted for implementation by the core team.
```
Comment (gaearon) [inline content/blog/2017-12-07-introducing-the-react-rfc-process.md]: Can it be implemented by somebody *outside* the team too?

Comment (gaearon) [reply content/blog/2017-12-07-introducing-the-react-rfc-process.md]: I guess “by” placement is ambiguous here.

Comment (acdlite) [reply content/blog/2017-12-07-introducing-the-react-rfc-process.md]: Yeah, I'll rephrase. (Though it's true that anyone is free to submit a PR and implement it.)

File: content/blog/2017-12-07-introducing-the-react-rfc-process.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+As with our other repositories, we do ask that you complete a [Contributor License Agreement](https://github.com/reactjs/rfcs#contributor-license-agreement-cla) before we can accept your PR.
+
+## What Types of Changes Should Be Submitted As RFCs?
+
+Generally, any idea that would benefit from additional review or design before being implemented is a good candidate for an RFC. As a rule of thumb, this means any proposal that adds, changes, or removes a React API.
+
+Not every change must go through the RFC process. Bug fixes or performance improvements that don't touch the API can be submitted directly to the main library.
+
+We now have several different repositories for contributing improvements to React:
```
Comment (clemmy) [inline content/blog/2017-12-07-introducing-the-react-rfc-process.md]: The meaning of this sentence is a little unclear to me. Do those repositories contribute improvements to React? Or are you able to contribute to React by contributing to of the mentioned repositories? I'm leaning towards the latter, but it took a few re-reads.

**Comment by alexkrolick** [comment]
Looking over the Context proposal, it's using Stage-2 ES and Flow typings, which are great, but for someone not familiar with Flow a bit hard to follow in parts. We've added syntax guidelines [to the docs]( https://github.com/reactjs/reactjs.org/blob/master/CONTRIBUTING.md#guidelines-for-code-examples) already, do you think something like that would be warranted for the RFCs, too?

File: content/blog/2017-12-07-introducing-the-react-rfc-process.md (hunk 2 above)
Comment (bvaughn) [reply content/blog/2017-12-07-introducing-the-react-rfc-process.md]: Not sure if the sentence would have stood out to me as much before seeing Clam's comment, but now it does.
